testing/matrix/linter/rules/rules.go

```diff
@@ -177,13 +197,35 @@ func containerImageTagLatest(object storage.StoreObject, containers []v1.Contain
 				"Can't parse an image for container",
 			)
 		}
-		if imageParts[1] == "latest" {
+
+		if imageParts[0] != defaultRegistry {
 			return errors.NewLintRuleError("CONTAINER003",
 				object.Identity()+"; container = "+c.Name,
 				nil,
-				"Image tag \"latest\" used",
+				"Image name should be "+defaultRegistry,
 			)
 		}
+
+		r := regexp.MustCompile("[A-Fa-f0-9]{64}")
```

Comment:
Compiling regex in a loop is not efficient. Consider moving it out of the loop/function.

Reply:
Moved, thanks !